readthedocs/projects/tasks/mixins.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import annotations
+
```

Comment:
I just noticed that this slipped in -- I've been working with `list.__class_getitem__` under Python 3.8 a little too long, I guess. :grin: 

I plan to remove it, since it should not be needed, but let me know if we'd prefer to have it.